docs/hello_nextflow/seqera/01_run_with_cli.md

```diff
@@ -0,0 +1,167 @@
+## 1. Use Seqera Platform to capture and monitor Nextflow jobs launched from the CLI
+
+We'll start by using the Nextflow CLI to launch a pipeline and monitor it in Seqera Platform.
+Start by logging into the [Seqera Platform](https://cloud.seqera.io/).
+
+!!! info "Nextflow Tower"
+
+    Seqera Platform was previously known as Nextflow Tower.
+    You'll still see references to the previous name in environment variable and cli option names.
+
+### 1.1. Set up your Seqera Platform token by exporting it to your environment
+
+Follow these steps to set up your token:
+
+1.  Create a new token by clicking on the **Settings** drop-down menu:
+
+    ![Create a token](seqera/img/usage_create_token.png)
+
+2.  Name your token:
+
+    ![Name your token](seqera/img/usage_name_token.png)
+
+3.  Save your token safely:
+
+    ![Save token](seqera/img/usage_token.png)
+
+    !!! note
+
+        Leave this browser tab open as we will need the token once more to store it as a Nextflow secret.
+
+4.  To make your token available to the Nextflow CLI, export it on the command line:
+
+    Open a terminal and type:
+
+    ```bash
+    export TOWER_ACCESS_TOKEN=eyxxxxxxxxxxxxxxxQ1ZTE=
+    ```
+
+    Where `eyxxxxxxxxxxxxxxxQ1ZTE=` is the token you have just created.
+
+### 1.2. Run Nextflow cli with Seqera Platform visualizing and capturing logs
+
+Run your Nextflow workflows as usual with the addition of the `-with-tower` command:
```

Comment:
'as usual with the addition of' is a bit ambiguous, would separate / clarify

Reply:
Fixed.